cpp/dolfinx/mesh/MeshTags.h

```diff
@@ -75,6 +75,26 @@ class MeshTags
   /// Move assignment
   MeshTags& operator=(MeshTags&& tags) = default;
 
+  /// Find all MeshTag indices of a given value, local-to-process
+  /// @param[in] i The value
+  /// @return The indices
+  std::vector<std::int32_t> find_indices(T i) const
```

Comment:
I would call argument `T value`, its more telling.

Reply:
The return type needs to be an Eigen type to avoid a copy when passing through to Python.

Reply:
@garth-wells We talked about not complicating return types with Eigen where not needed? Would be good if we find a way how to return py::array_t and make sure the underlying std::vector is moved.

Reply:
One thing we could do, is to not expose this function to python, as in python it is quite easy to get the tags. If we only expose this in C++, it can be kept as a list.

Reply:
Good point - can just use NumPy.